libs/libcommon/src/libcommon/utils.py

```diff
@@ -91,6 +91,12 @@ class FeatureItem(TypedDict):
     type: Row
 
 
+class PaginatedResponse(TypedDict):
+    features: List[FeatureItem]
+    rows: Any
+    num_total_items: int
```

Comment:
to keep it coherent with what we have on DatasetSize, I would rename it to

```suggestion
    num_total_rows: int
```